Update all dependencies & Readme, Add windi dev tools, visibility actions button style improvement, add 1.19

AlessioGr opened the pull request:
The windi dev tools import shoould be empty in production.

Didn't notice any issues after updating the dependencies.

My IDE did some other changes automagically, they don't seem to be bad, though.

MiniDigger commented:
Does windi dev tools actually work? Didn't when I last tried.
When I updated deps last week I ran into really weird oom issues the day after so I just reverted, I'll look tomorrow if it's better now

AlessioGr commented:
> Does windi dev tools actually work? Didn't when I last tried.
> When I updated deps last week I ran into really weird oom issues the day after so I just reverted, I'll look tomorrow if it's better now

Yep, the dev tools work perfectly!

Didn't notice any dependency issues either apart from vite-plugin-eslint and typescript, so I left these unupdated.

MiniDigger commented on frontend-new/src/components/layout/Header.vue:
```diff
@@ -150,38 +150,41 @@ function isRecent(date: string): boolean {
-                </router-link>
-              </div>
+            <!-- dummy diff to make the transition work on pages where template root has multiple elements -->
+            <div id="#page">
```
dont duplicate the ids

MiniDigger replied:
fwiw, I only named it page in that one case because its nice to see it in the dev tools, its meanlingless other than that
